lib/blueberry_rails/version.rb

```diff
@@ -3,5 +3,5 @@ module BlueberryRails
 
   RUBY_VERSION = '2.1.5'
 
-  RAILS_VERSION = '4.1.9'
+  RAILS_VERSION = '4.2.0'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.